src/components/Common/StatementComponent.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
```

Comment:
Why the name `StatementComponent.js` and not just `Statement.js` ?

Reply:
idk, make sure everyone knows its a component ahah